op-node/cmd/genesis/cmd.go

```diff
@@ -187,10 +189,23 @@ var Subcommands = cli.Commands{
 				}
 
 				if config.L1StartingBlockTag == nil {
-					l1StartBlock, err = client.BlockByNumber(context.Background(), nil)
+					// Retrieve SystemConfig.startBlock()
+					callOpts := &bind.CallOpts{Context: context.Background()}
+					systemConfig, err := bindings.NewSystemConfig(config.SystemConfigProxy, client)
 					if err != nil {
-						return fmt.Errorf("cannot fetch latest block: %w", err)
+						return fmt.Errorf("cannot create instance of SystemConfig contract: %w", err)
 					}
+
+					l1StartBlockNum, err := systemConfig.StartBlock(callOpts)
```

Comment:
Do we have any automated tests for the `op-node genesis l2` command? What is the easiest way to check if this code is working properly?

Reply:
The devnet uses this command

Reply:
Ok nice. I will rely on the automated devnet to test this code then